sphinx/source/docs/releases/2.1.0.rst

```diff
@@ -3,23 +3,31 @@
 2.1.0
 =====
 
-Bokeh Version ``2.1.0`` (May/June 2020) is a minor-release.
+Bokeh Version ``2.1.0`` (June 2020) is a minor-release.
+
+* New scatter marker types (:bokeh-issue:`9863`, :bokeh-issue:`9890`)
+* Performance improvements (:bokeh-issue:`9515`, :bokeh-issue:`10006`, :bokeh-issue:`10021`, :bokeh-issue:`10072`)
+* ``push_notebook`` fixed for Google Colaboratory (:bokeh-issue:`9302`)
```

Comment:
It might be worth noting that the comm support in Colab is not yet generally available. I'll ping them to get a clearer timeline.